lib/ebaze/accounts.ex

```diff
@@ -0,0 +1,104 @@
+defmodule Ebaze.Accounts do
+  @moduledoc """
+  The Accounts context.
+  """
+
+  import Ecto.Query, warn: false
+  alias Ebaze.Repo
+
+  alias Ebaze.Accounts.User
+
+  @doc """
```

Comment:
Could you remove the doc junk?

Reply:
55f3d6f71eeb432810fefae5ec3902a76a216210